roles/openshift_manage_node/tasks/main.yml

```diff
@@ -7,15 +7,15 @@
   delay: 5
   with_items: openshift_nodes
 
-- name: Handle unscheduleable node
+- name: Handle unschedulable node
   command: >
     {{ openshift.common.admin_binary }} manage-node {{ item }} --schedulable=false
```

Comment:
If updating it as a node fact like the comment above, you can shorten these two tasks to the following:
```
name: Set node schedulability
command: > 
  {{ openshift.common.admin_binary }} manage-node {{ item }} --schedulable={{ 'true' if hostvars[item].node.schedulable | bool else 'false' }}
```